File: BackEnd_Learn/Java/jwt/src/main/java/com/example/jwt/Security/Jwt/JwtConfigurer.java
package com.example.jwt.Security.Jwt;

import lombok.RequiredArgsConstructor;
import org.springframework.security.config.annotation.SecurityConfigurerAdapter;
import org.springframework.security.config.annotation.web.builders.HttpSecurity;
import org.springframework.security.web.DefaultSecurityFilterChain;
import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;

@RequiredArgsConstructor
public class JwtConfigurer extends SecurityConfigurerAdapter<DefaultSecurityFilterChain, HttpSecurity> {
    private final JwtTokenProvider jwtTokenProvider;

    @Override
    public void configure(HttpSecurity httpSecurity) {
        JwtTokenFilter tokenFilter = new JwtTokenFilter(jwtTokenProvider);
        httpSecurity.addFilterBefore(tokenFilter, UsernamePasswordAuthenticationFilter.class);
    }
}

File: BackEnd_Learn/kotlin/entity/src/main/kotlin/com/example/entity/security/jwt/JwtConfigurer.kt
package com.example.entity.security.jwt

import org.springframework.security.config.annotation.SecurityConfigurerAdapter
import org.springframework.security.config.annotation.web.builders.HttpSecurity
import org.springframework.security.web.DefaultSecurityFilterChain
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

class JwtConfigurer(val jwtTokenProvider: JwtTokenProvider): SecurityConfigurerAdapter<DefaultSecurityFilterChain, HttpSecurity>(), WebMvcConfigurer {
    override fun configure(builder: HttpSecurity?) {

    }
}